SC_redesign/Crop_and_Soil/soil/carbon_cycling/residue_partition.py

```diff
@@ -160,3 +162,142 @@ def _determine_plant_metabolic_active_carbon_usage(decomposition_moisture_effect
+
+    @staticmethod
+    def _determine_plant_structural_to_slow_or_active_rate(plant_residue_metabolic_fraction: float,
+                                                           structural_decomposition_factor=0.076) -> float:
+        # TODO: check with subject expert for structural_decomposition_factor's default value. issue #428
+        """This method calculates the rate at which above ground structural carbon decomposes into slow or active carbon
+
+        Parameters
+        ----------
+        structural_decomposition_factor: float, default = 0.076
+            structural decomposition factor (unitless)
+        plant_residue_metabolic_fraction: float
+            fraction of plant residue that is metabolic (unitless)
+        Returns
+        -------
+        float
+            the rate at which above ground structural carbon decomposes into slow or active carbon (unitless)
+
+        References
+        -------
+        pseudocode_soil S.6.B.I.9
+        """
+        # TODO: contradiction with the equation in pseudocode_soil. issue #427
+        return structural_decomposition_factor * math.exp(-3) * 1 - plant_residue_metabolic_fraction
+
+    @staticmethod
+    def _determine_plant_structural_to_slow_active_carbon_amount(plant_structural_to_slow_or_active_rate: float,
+                                                                 decomposition_moisture_effect: float,
+                                                                 decomposition_temperature_effect: float,
+                                                                 plant_structural_carbon_amount: float) -> float:
+        """This methods determines the amount of plant structural carbon decomposed into slow or active carbon
+
+        Parameters
+        ----------
+        plant_structural_to_slow_or_active_rate: float
+            rate at which above ground structural carbon decomposes into slow or active carbon (unitless)
+        decomposition_moisture_effect: float
+            moisture effect on decomposition factor (unitless)
+        decomposition_temperature_effect: float
+            temperature effect on decomposition factor (unitless)
+        plant_structural_carbon_amount: float
+            pant structural carbon amount(kg/ha)
+
+        Returns
+        -------
+        float
+            amount of plant structural carbon decomposed into slow or active carbon (kg/ha)
+
+        References
+        -------
+        pseudocode_soil S.6.B.I.10
+        """
+        return plant_structural_to_slow_or_active_rate * decomposition_moisture_effect \
+            * decomposition_temperature_effect\
+            * plant_structural_carbon_amount
+
+    @staticmethod
+    def _determine_structural_carbon_transfer_amount(plant_structural_carbon_amount: float,
+                                                     tillage_fraction: float) -> float:
+        """Determines the amount of transfer of structural carbon during tillage
```

Comment:
```suggestion
        """Determines the amount to transfer from structural carbon to <other carbon type?> during tillage
```
I think this is what this method is determining, but ignore if I am wrong